tooling/validateSBOMcontent.sh

```diff
@@ -22,6 +22,8 @@ SBOMFILE="$1"
 MAJORVERSION="$2"
 EXPECTED_SCM_REF="$3"
 
+[ "${EXPECTED_SCM_REF}" = "scm_not_specified" ] && EXPECTED_SCM_REF="null"
```

Comment:
Not keen on having special "strings", best just allow blank value if not specified